src/PresentationalComponents/ZeroState/zeroStateConstants.js

```diff
@@ -1,28 +1,96 @@
+            link: 'https://access.redhat.com/documentation/en-us/red_hat_hybrid_cloud_console/2023/html/configuring_notifications_and_integrations_on_the_red_hat_hybrid_cloud_console/index'
+        }
+    ]
 };
 
 const COMPLIANCE_ZERO_STATE = {
     header: {},
-    otherApps: {}
+    otherApps: {},
+    documentation: [
+        {
+            title: 'Assessing and Monitoring Security Policy Compliance',
+            link: 'https://access.redhat.com/documentation/en-us/red_hat_insights/2023/html/assessing_and_monitoring_security_policy_compliance_of_rhel_systems'
+        },
+        {
+            title: 'Generating Compliance Service Reports',
+            link: 'https://access.redhat.com/documentation/en-us/red_hat_insights/2023/html/generating_compliance_service_reports'
+        },
+        {
+            title: 'Insights Compliance - Supported configurations',
+            link: 'https://access.redhat.com/articles/6644131'
+        },
+        {
+            title: 'Compliance APIs',
+            link: 'https://console.redhat.com/docs/api/compliance'
+        },
+        {
+            title: 'Configuring notifications & integration',
+            link: 'https://access.redhat.com/documentation/en-us/red_hat_hybrid_cloud_console/2023/html/configuring_notifications_and_integrations_on_the_red_hat_hybrid_cloud_console/index'
+        }
+    ]
 };
 
 const DRIFT_ZERO_STATE = {
     header: {},
-    otherApps: {}
+    otherApps: {},
+    documentation: [
+        {
+            title: 'Comparing System Configurations and Baselines',
+            link: 'https://access.redhat.com/documentation/en-us/red_hat_insights/2023'
+        },
+        {
+            title: 'Drift - Comparison API',
+            link: 'https://console.redhat.com/docs/api/drift'
+        },
+        {
+            title: 'Drift - Baseline API',
+            link: 'https://console.redhat.com/docs/api/system-baseline'
+        },
+        {
+            title: 'Configuring notifications & integration',
+            link: 'https://access.redhat.com/documentation/en-us/red_hat_hybrid_cloud_console/2023/html/configuring_notifications_and_integrations_on_the_red_hat_hybrid_cloud_console/index'
+        }
+    ]
 };
 
 const INSIGHTS_ZERO_STATE = {
     header: {
         description: messages.insightsDescription,
         //An array like this would be passed into the app
         commands: [
-            { intructions: 'Install the client on the RHEL system', command: '[root@server] testing install insights-clint' },
-            { intructions: 'Install the client on the RHEL system', command: '[root@server] testing install insights-clint' },
-            { intructions: 'Install the client on the RHEL system', command: '[root@server] testing install insights-clint' }
+            {
+                intructions: 'Install the client on the RHEL system',
+                command: '[root@server] testing install insights-clint'
```

Comment:
`[root@server]` needs to be deleted from all commands above and below as it is part of a terminal prompt message.

Reply:
They actually include this in the mock-ups: https://www.sketch.com/s/550bd962-ca1f-42d8-8d89-d457527a059a/a/qbe9Ol2#Inspect

Reply:
@mkholjuraev ^

Reply:
When the the command is copied, it will copy `[root@server]` also. This will cause the command not to work with an error. Mockup also needs a change or we need to find a way to copy the correct portion of the command which is `insights-client --register`.

Reply:
This is sort of a temporary placeholder. The copy for dashboards page hasnt been finalized yet, which is one of the reasons its moved to P3. Since this component will not be displayed in dashboard until P3, I kept it as is. Once mocks are updated on the google doc and I get the notification from stefan K, this will be updated and dashboard will adopt zero state. As of now, no app will display this